app/src/main/res/values/strings.xml

```diff
@@ -108,4 +109,11 @@
         After all, the internet shouldn\'t feel so creepy, and getting the privacy you deserve online should be as simple closing the blinds.</string>
     <string name="aboutMoreLink">More at duckduckgo.com/about</string>
 
+    <!-- Bookmarks Activity -->
+    <string name="boomarksMenuTitle">Bookmarks</string>
+    <string name="bookmarksActivityTitle">Bookmarks</string>
+    <string name="deleteBookmarkContentDescription">Delete bookmark for %s</string>
+    <string name="bookmarkDeleteConfirmMessage">Are you sure you want to delete this bookmark?\n\n\'%s\'</string>
```

Comment:
Not sure if in design, but it would be nice to be able to see the URL somewhere. maybe in the deletion dialog. not urgent, though

Reply:
Yeah, I even had the URL in the list as well at one point but it's not in the design or on iOS either so I figured if title was good enough for the list, it's probably ok for this.  Plus this is temporary until we implement the more advance book marks UI where it will just be deleted instantly and then show a snackbar with an undo button.